.github/workflows/monday_connector_unittests.yaml

```diff
@@ -0,0 +1,32 @@
+name: unit tests for monday_connector 
+
+on:
+  push:
+    branches:
+      - pipeline-test
+    paths: 
+      - monday_connector/**
+      
+  pull_request:
+
+jobs:
+  build:
```

Comment:
This isn't a build step. It is a 'unittest' step. This text will show as 'build' in the step automation and it should be consistent with what is happening.